src/PointwiseFunctions/AnalyticSolutions/WaveEquation/PlaneWave.cpp

```diff
@@ -79,6 +79,34 @@ tnsr::ii<T, Dim> PlaneWave<Dim>::d2psi_dxdx(const tnsr::I<T, Dim>& x,
   return result;
 }
 
+template <size_t Dim>
+Variables<tmpl::list<ScalarWave::Pi, ScalarWave::Phi<Dim>, ScalarWave::Psi>>
+PlaneWave<Dim>::evolution_variables(const tnsr::I<DataVector, Dim>& x,
+                                    double t) const noexcept {
+  Variables<tmpl::list<ScalarWave::Pi, ScalarWave::Phi<Dim>, ScalarWave::Psi>>
+      variables{x.begin()->size()};
+  get<ScalarWave::Psi>(variables) = psi(x, t);
```

Comment:
This (and the time derivatives) could be done much more efficiently if you didn't call the other member functions

Reply:
Yes, but that would lead to a fair bit of code duplication, and I can't see us having computing analytic solutions for scalar waves being something where code duplication is acceptable for increased performance. Thoughts?

Reply:
So I won't hold up this PR for this, but there should be an issue to think about whether we can have an efficient implementation that avoids code duplication

Reply:
Reported issue in #489